### ISSUE TITLE
Android: track Performance Class of the device

### ISSUE THREAD

**Issue by yogurtearl** [getsentry/sentry-java#3863]
### Problem Statement

For a given issue/trace I want to know the performance class of the device. 

https://developer.android.com/topic/performance/performance-class

Also, I would like to be able to slice performance data by performance class. 

### Solution Brainstorm

Incorporate performance class data from `androidx.core:core-performance` and `androidx.core:core-performance-play-services` 

see https://developer.android.com/topic/performance/performance-class

**Comment by romtsn** [comment]
hi @yogurtearl we already expose the [device.class](https://docs.sentry.io/concepts/search/searchable-properties/#device-classification) property for searching errors/transactions/spans, would it be enough or you want the google play perf class specifically?

**Comment by yogurtearl** [comment]
Using the standardized Google Play device classes would be more useful. 

They have a lot more details on what it means to be in each class.

**Comment by romtsn** [comment]
makes sense! We'd have to make it a separate artifact since it pulls in a new dependency, but I think we can make it work, thanks for the suggestion 👍

**Comment by markushi** [comment]
@yogurtearl thanks for the suggestion! We've added it to our backlog. In the meantime you could utilize `Sentry.setTag("play-device-class", <>)` to manually supply this information.
